src/action/Link/Link.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import { heading1 } from "src/guidelines/theme";
+import PlusIcon from "src/icons/plus";
+import { sizing } from "src/utils/utils";
+import Link from "./Link";
+
+const meta: Meta<typeof Link> = {
+  title: "action/Link",
+  component: Link,
+  argTypes: {
+    leadingIcon: {
+      control: "boolean",
+    },
+    trailingIcon: {
+      control: "boolean",
+    },
+    underline: {
+      control: { type: "select", options: ["always", "hovered", "never"] },
+    },
+  },
+  args: {
+    children: "this is a link",
+    href: "#",
+    isDisabled: false,
+    target: "_blank",
```

Comment:
When "target" is set to blank, it will be nice to have an error for "rel" "noopener noreferrer"